### PR TITLE
Akaiko/i18n editor setup it

### PR DESCRIPTION
**Pull request by akaik0**
#### What kind of changes does this PR include?

- Translated content

#### Description

I added the translation of the page "Editor set up" to Italian

### CODE REVIEW

**Comment by milovangudelj** [comment]
When translating make sure to also update internal links like `/en/get-started -> /it/get-started` and keep in mind that each section link (those starting with an hash tag) also get automatically translated based on the section heading. (e.g. `#other-code-editors -> #altri-editor-di-testo` since the section title is "Altri Editor di Testo")

File: src/content/docs/it/editor-setup.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add --dev prettier prettier-plugin-astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Prettier rivelerà automaticamente il plugin e lo userà per processare i file `.astro` quando li esegui:
```
Comment (ematipico) [inline src/content/docs/it/editor-setup.mdx:88]:
```suggestion
Prettier rivelerà automaticamente il plugin e lo userà per processare i file `.astro` quando esegui:
```

File: src/content/docs/it/editor-setup.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+```shell
+prettier --write .
+```
+
+Vedi [README del plugin di Prettier](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md) per maggiori informazionisulle opzioni supportate, come configurare Prettier all'interno di VS Code, e altro.
```
Comment (ematipico) [inline src/content/docs/it/editor-setup.mdx:94]:
```suggestion
Vedi [README del plugin di Prettier](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md) per maggiori informazioni sulle opzioni supportate, come configurare Prettier all'interno di VS Code, e altro.
```

**Review by ematipico — APPROVED** [review]
Looks good! Just left few suggestions

File: src/content/docs/it/editor-setup.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Per cominciare, installa [l'estensione Astro VS Code ](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) oggi.
+
+📚 Guarda come [configurare TypeScript](/it/guides/typescript/) nel tuo progetto Astro.
+
+## Gli IDE di JetBrains 
+
+Il supporto iniziale per Astro è arrivato da WebStorm a Gennaio 2023. Puoi installare il plugin ufficiale attraverso [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) oppure cercando "Astro" nella tab dei plugin dell'IDE. Questo plugin include proprietà come l'evidenziazione della sintassi, il completamento del codice e la formattazione, e in futuro ha intenzione di aggiungere anche funzionalità più avanzate. Il plugin è disponibile anche per tutti gli altri [JetBrains IDEs con il supporto di JavaScript](https://www.jetbrains.com/products/#lang=js&type=ide).
+
+Il futuro [Fleet IDE](https://www.jetbrains.com/fleet/) di JetBrains include il supporto del server di lingua  e sarà in grado di usare gli strumenti ufficiali di Astro.
```
[the remark this answers is not in the document]
Comment (akaik0) [reply src/content/docs/it/editor-setup.mdx:29]: So how am I supposed to write this sentence? I mean should I just leave "language server"? Or LSP. Sorry but I'm not too skilled with translations of this type.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @akaik0 🙌

**Pull request merged**